Kernel Containerized Performance Tests: Print run-cyclictest output

dylan-matthew-turner opened the pull request:
## Reason

The script `run-cyclictest` is run from within a container, and although the output is logged, it is never printed, meaning the tests do not print the PASS/FAIL messages required of ptests.

## Implementation

- After tests run, print the run-cyclictest log to stdout
- Remove date from log name (unneeded and makes it complicated to correctly cat the log)

## Testing

Ran through on ATS hardware and saw PASS messages appear

mike-petersen-ni commented on recipes-kernel/kernel-tests/kernel-containerized-performance-tests-files/test_kernel_cyclictest_fio_containerized.sh:
```diff
@@ -37,9 +37,11 @@ RESULT=$(docker run --privileged --network=host \
         | tr -d '\r' | tr -d '\n')
 
+# Make sure we print the PASS/FAIL message
+cat ${LOG_DIR}/run_cyclictest-`date +'%Y_%m_%d-%H_%M_%S'`-fio_containerized.log
```
You're recalculating the timestamp, so if you're unlucky with the clock, these files may not exist.

dylan-matthew-turner replied:
I guess we don't really need the timestamp anyway since the target is wiped every run

dylan-matthew-turner commented:
Changes:

- Removed timestamp from log name since target is rewritten anyway

dylan-matthew-turner committed:
k-cont-perf-tests: print run-cyclictest output

run-cyclictest is run from a container, and although the output is
logged, it is never printed, meaning the tests do not print the
PASS/FAIL messages required of ptests

Print the logs to have this output.

Signed-off-by: Dylan Turner <dylan.turner@ni.com>

dylan-matthew-turner commented:
Changes:

- Correct signed-off email

chaitu236 reviewed (changes requested):
Please describe the testing you did in description. And add work item number.

dylan-matthew-turner commented:
I hadn't done testing bc it is a small change, but I will set something up and get back

dylan-matthew-turner commented:
@chaitu236 Added
